### PR TITLE
tag: Restrict PNT posting for "not English" and "rough translation" to mainspace

### LINKED ISSUE #604: Block posting to [[WP:PNT]] on ineligible articles aka drafts
Remove the ability to post to WP:PNT when tagging as not English per https://en.wikipedia.org/wiki/User_talk:RhinosF1#WP:PNT when on pages that shouldn't be added to PNT

### PR DESCRIPTION
**Pull request by DannyS712**
Closes #604

### CODE REVIEW

**Comment by siddharthvp** [comment]
I can handle the merge conflict, thanks, but @DannyS712 the `translationLanguage` input should be available even in draftspace, since it's added along with the tag. And the templates on-wiki should be edited to not show the "please post to PNT" part when used on draftspace.

**Comment by DannyS712** [comment]
@siddharthvp done, I think

File: modules/friendlytag.js [hunk 1]
```diff
@@ -342,43 +342,45 @@ Twinkle.tag.updateSortOrder = function(e) {
...
+							type: 'checkbox',
+							list: [
+								{
+									label: 'Notify article creator',
+									checked: true,
+									tooltip: "Places {{uw-notenglish}} on the creator's talk page."
+								}
+							]
+						});
+					};
```
Comment (Amorymeltzer) [inline modules/friendlytag.js]: No semi colon here
```suggestion
					}
```

**Comment by Amorymeltzer** [comment]
Agreed that the templates should be edited before this goes forward.

**Comment by DannyS712** [comment]
I'm not sure what template is on-wiki that needs to be changed...

**Comment by Amorymeltzer** [comment]
The templates in question.  `{{not English}}` and `{{rough translation}}` both make mention of posting at PNT.  If these are to be used on Drafts, that should certainly not show on drafts, regardless of this.

**Comment by DannyS712** [comment]
Are you saying that the _options_ shouldn't be shown on drafts, or that the mentions of posting at  PNT _in the template itself_ should be hidden when it is transcluded on a draft page?

**Comment by Amorymeltzer** [comment]
The latter; you've done the former here, no? :wink:

**Comment by DannyS712** [comment]
@Amorymeltzer done, see https://w.wiki/3Wx and https://w.wiki/3Wy

**Comment by Amorymeltzer** [comment]
Oh and with the fancy short urls!

**Comment by RhinosF1** [comment]
Thanks all!!

**Pull request merged**